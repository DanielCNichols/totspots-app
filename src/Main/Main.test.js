import React from 'react';
import ReactDOM from 'react-dom';
import Main from './Main.js';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Main />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders Main as expected', () => {
  const tree = renderer.create(<Main name='Main' />).toJSON();
  expect(tree).toMatchSnapshot();
});
