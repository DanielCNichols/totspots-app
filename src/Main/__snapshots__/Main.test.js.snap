// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Main as expected 1`] = `
<div
  className="main"
/>
`;
